examples/foundationdb/README.md

```diff
@@ -25,6 +25,22 @@ on you cluster.
 * Kubernetes 1.9+ with Beta APIs enabled.
 * PV support on the underlying infrastructure.
 * Kanister version 0.92.0 with `profiles.cr.kanister.io` CRD installed.
+* Docker CLI installed
+* A docker image containing the required tools to back up FoundationDB.
```

Comment:
```suggestion
* A docker image containing the required tools to back up FoundationDB.

```
The next sentence is added as a new bullet point in the preview. Add a newline if you don't want to add it as a bullet point.

Reply:
Looks okay to me 👀 
![image](https://github.com/kanisterio/kanister/assets/6504783/36e458e1-c975-421c-b972-28b3d19c0531)

Reply:
I meant, is it intended to be a separate point or are both sentences part of the same point

Reply:
I think we should keep that as separate point.